### PR TITLE
GitHub task connector

### PR DESCRIPTION
**Pull request by BeckerFrank**
See Issue https://github.com/eclipse-mylyn/org.eclipse.mylyn/issues/18 this replaces [GitHub Pull request 38](https://github.com/eclipse-mylyn/org.eclipse.mylyn/pull/38)

### CODE REVIEW

File: org.eclipse.mylyn-target/mylyn-e4.26.target [hunk 1]
```diff
@@ -12,7 +12,7 @@
 			<unit id="org.eclipse.sdk.feature.group" version="0.0.0"/>
 		</location>
 		<location includeAllPlatforms="false" includeConfigurePhase="true" includeMode="planner" includeSource="true" type="InstallableUnit">
-			<repository location="https://download.eclipse.org/egit/updates-6.4/"/>
+			<repository location="https://download.eclipse.org/egit/updates-6.2/"/>
```
Comment (ruspl-afed) [inline org.eclipse.mylyn-target/mylyn-e4.26.target:15]: Why do we need to degrade?

File: mylyn.github/pom.xml [hunk 2]
```diff
@@ -0,0 +1,416 @@
...
+  <groupId>org.eclipse.mylyn.github</groupId>
+  <artifactId>github-parent</artifactId>
+  <version>6.2.0-SNAPSHOT</version>
+  <packaging>pom</packaging>
+
+  <name>Eclipse EGit Mylyn GitHub Connector Parent</name>
+
+  <mailingLists>
+    <mailingList>
+      <name>egit-dev Mailing List</name>
```
Comment (ruspl-afed) [inline mylyn.github/pom.xml:28]: Should we switch mail list?

File: mylyn.github/pom.xml [hunk 3]
```diff
@@ -0,0 +1,416 @@
...
+      <post>egit-dev@eclipse.org</post>
+      <subscribe>https://dev.eclipse.org/mailman/listinfo/egit-dev</subscribe>
+      <unsubscribe>https://dev.eclipse.org/mailman/listinfo/egit-dev</unsubscribe>
+      <archive>http://dev.eclipse.org/mhonarc/lists/egit-dev</archive>
+    </mailingList>
+  </mailingLists>
+
+  <issueManagement>
+    <url>https://bugs.eclipse.org/bugs/buglist.cgi?query_format=advanced;product=EGit;component=GitHub</url>
```
Comment (ruspl-afed) [inline mylyn.github/pom.xml:37]: Should we switch issue management?

**Review by ruspl-afed — COMMENTED** [review]
Do we plan to include it to the build?

**Comment by ruspl-afed** [comment]
How can we solve eclipsefdn/eca failure?

File: mylyn.github/pom.xml (hunk 2 above)
Comment (wimjongman) [reply mylyn.github/pom.xml]: Yes

**Comment by wimjongman** [comment]
> How can we solve eclipsefdn/eca failure?

We have imported existing code from the Egit repo and pushing it now in our repo. With this push, we get an ECA check on an email address from Thomas Wolf.

I have sent a mail to Thomas to ask if he can still sign the ECA with that address. If not, we will accept the code because it comes from an existing valid Eclipse repository, and we still have the history. @waynebeaton Wayne, can you please comment if this is correct?

**Comment by BeckerFrank** [comment]
> Do we plan to include it to the build?

Yes, I work on tis and hope to finish this soon.

File: org.eclipse.mylyn-target/mylyn-e4.26.target (hunk 1 above)
Comment (BeckerFrank) [reply org.eclipse.mylyn-target/mylyn-e4.26.target:15]: I have fixed this. wait for the next commit.

File: mylyn.github/pom.xml (hunk 2 above)
Comment (BeckerFrank) [reply mylyn.github/pom.xml]: I also changed this and include the GitHub Connector in the maven build and the updatesite.

**Comment by waynebeaton** [comment]
Thomas Wolf is a current committer on the Eclipse eGit project.  AFAICT all of the current commits attributed to Thomas were made during the period in which he held committer status. Based on the history of his Eclipse account, he changed email addresses sometime since creating those commits. This is why you (and the system) can't find an ECA.

So... you've encountered a hole in the ECA system. I'm pretty sure that we already have an issue open for this. I'll look harder when I have a chance later.

In the meantime, the ECA checker is just warning you of a problem that I'm very confident is bogus, but should not be stopping you. Just merge the commits.

**Comment by BeckerFrank** [comment]
@wimjongman, @ruspl-afed: What is needed to continue here?

**Review by wimjongman — APPROVED** [review]
It is not perfect yet but I think we should take this and then work on the remaining glitch.

**Pull request merged**